src/theory/quantifiers/instantiate.cpp

```diff
@@ -264,9 +264,8 @@ bool Instantiate::addInstantiation(
     return false;
   }
 
-  d_total_inst_debug[q]++;
+  d_total_inst_debug[q] = d_total_inst_debug[q] + 1;
```

Comment:
Why this change?

Reply:
because its context-dependent now, which doesnt support `++`.